prelude/cxx/cxx_library.bzl

```diff
@@ -667,7 +735,7 @@ def cxx_library_parameterized(ctx: AnalysisContext, impl_params: CxxRuleConstruc
     # Propagate link info provider.
     if impl_params.generate_providers.merged_native_link_info or impl_params.generate_providers.template_placeholders:
         # Gather link inputs.
-        inherited_non_exported_link = cxx_inherited_link_info(non_exported_deps)
+        inherited_non_exported_link = cxx_inherited_link_info(non_exported_deps + filter(None, [ctx.attrs.precompiled_header if compiler_type != "clang" else None]))
```

Comment:
Why only if not clang? Please comment/document.

Reply:
inverted and switched to windows, also added a comment about windows needing the linkables